src/bic/model/interaction.js

```diff
@@ -244,11 +243,29 @@ The argument change event.
 
               resolve(model);
             } else {
-              model.defaultView(app.interactions.models, app.siteVars);
-              resolve(model);
+              if (app.has('homeScreen') && app.get('homeScreen') !== false && app.has('homeInteraction')) {
```

Comment:
I see this exact condition pop up in 3 different `if` statements. Maybe there should be a new shared `app#hasHomeInteraction()` method that gets used instead? That'll help with future maintenance, as we won't have to update 3 different `if` statements later.